public-ui/src/app/shared/components/rh-header/rh-header.component.css

```diff
@@ -243,18 +241,17 @@ li.flip::before, li.flip::before {
   font-family: 'GothamBook', 'Gotham B', 'Helvetica Neue', sans-serif;
   font-weight: initial;
   text-transform: capitalize!important;
-} 
-
+}
 
-.site-nav-container nav.main-navigation .menu-primary-container ul.main-menu li ul.sub-menu,
-.sub-menu li ul {
+.site-nav-container nav.main-navigation .menu-primary-container ul.main-menu li ul.sub-menu, .sub-menu li ul {
   list-style: none;
 }
 
 .site-nav-container nav.main-navigation .menu-primary-container>ul.main-menu>li>ul.sub-menu {
   list-style: none;
   padding-left: 1rem;
-  display: none;
+  top: -10000px;
+  position: absolute;
```

Comment:
the first state of the dropdown menu is off the page. This allows the links to be tabbed through because it never actually disappears from the DOM.